gallery/gallery-war/src/main/webapp/WEB-INF/tags/silverpeas/gallery/editMedia.tag

```diff
@@ -180,6 +182,7 @@
         errorMsg += "<li>" + error.message + "</li>";
         errorNb++;
       });
+      errorPromise = verifyFileInput($media);
```

Comment:
With the modification above, this code can be now:
```javascript
errorPromises.push(verifyFileInpyt($media));
```
See below my next comment to know how to use this array of promises.